ci/config/versions.yaml

```
@@ -0,0 +1,85 @@
+  - manifests/post-installation/ovn-template.yaml
+  - manifests/post-installation/flannel-template.yaml
+  - manifests/post-installation/hbn-template.yaml
+  - manifests/post-installation/blueman-template.yaml
+  - manifests/post-installation/dts-template.yaml
+  - manifests/dpf-installation/dpfoperatorconfig.yaml
+
+# Documentation sources
+documentation:
+  rdg_url: "https://docs.nvidia.com/networking/display/public/sol/rdg+for+dpf+with+ovn-kubernetes+and+hbn+services"
+  doca_platform_docs: "https://github.com/NVIDIA/doca-platform/tree/{branch}/docs"
+  hbn_ovn_folder: "examples/dpf"
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)